apps/aecore/src/aec_aens_governance.erl

```diff
@@ -0,0 +1,66 @@
+%%%-------------------------------------------------------------------
+%%% @doc Placeholder for functions instrumenting Naming auctions
+%%%
+%%% @end
+%%%-------------------------------------------------------------------
+-module(aec_aens_governance).
```

Comment:
I don't like the idea of spreading out governance in multiple modules.... hard to get a feeling what is under governance and what is not.

Reply:
I followed up here on what we did with coinbase inflation function. I guess it was for code clarity. Agree with you that holding everything in single governance module may be bigger hint for newcomer. I am not attached to the new module at all, if you think it's clearer to bring it to home governance, I am all up for it. Let me know.